infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServerNode.java

```diff
@@ -115,53 +106,22 @@ public void disable() {
      */
     @Override
     public void close() {
```

Comment:
this method is now doing the same thing as disable().  Can we remove one of them?

Reply:
I am keeping it for now because the comment indicates that when netty is the transport layer, disable() does something different.
Will need it when we bring back netty. But I have added a TODO